content/cli/cloud/secret.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Cloud Secrets
+menu: Secret
+weight: 40
+---
+
+Fluvio cloud secrets are set via the cli.  Each secret is a named value with all secrets sharing a namespace per account. Connector configuration files can refer to secrets by name, and the cloud connector infrastructure will provision the connector with the named secrets.
+
+Due to security concerns, listing actual secret values or downloading them after they have been set is not allowed. However, a listing of secret names as well as what date they were last set is accessible.
+
+## `fluvio cloud secret` subcommands
+
+The secrets cli is an added subcommand fluvio cloud as 'fluvio cloud secret'.
+
+Actions possible with a fluvio cloud secret are:
+- set
+- delete
+- list
+
+```sh
+fluvio cloud secret set --connector <NAME> <VALUE>
+fluvio cloud secret list
+fluvio cloud secret delete <NAME>
+```
+
+## `fluvio cloud secret set`
+
+Setting a scret of `<NAME>` will allow it to be refrenced by that name in connector configuration parameters that can use secret references.
+
+```
+fluvio cloud secret set --connector <NAME> <VALUE>
+fluvio cloud secret set -c <NAME> <VALUE>
+```
+All secrets are in a shared connector namespace, but a specific connector is only given access to secrects named in the configuration file of the connector.
+
+## `fluvio cloud secret list`
+
+`fluvio cloud secret list` will list only the secret names and their last update time. Once a secret has been set into fluvio cloud, it is stored so only referencing connectors may access the secret. There is no way to retreive the secret value from fluvio cloud.
+
+```sh
+$ fluvio cloud secret list
+SecretNames          LastUpdate
+CAT_FACTS_CLIENT_ID  12-10-2022 1:07pm
+CAT_FACTS_SECRET     01-02-2023 12:01am
+```
+
+
+## `fluvio cloud secret delete`
+
+This will delete the named secret.
+
+```
+fluvio cloud secret delete <NAME>
+```
+
+
+## Connector config file references
+
+The connector config files can reference cloud secrets by NAME generically as follows:
+
+```yaml
```

Comment:
We also need to explain how these secrets are used. Can you provide sample code or something that shows how this is useful?

Reply:
IMO, an explanation for how secrets are used with connectors do not belong in this section. This is a CLI reference.

As a user, I would not be in this section to discover how secrets work with connectors. It will also be easier for us to maintain by keeping the context focused on CLI usage, and then linking elsewhere when we want to introduce any kind of integration behavior